ext/sodium/libsodium.stub.php

```diff
@@ -165,75 +230,116 @@ function sodium_crypto_secretstream_xchacha20poly1305_pull(string &$state, strin
 function sodium_crypto_secretstream_xchacha20poly1305_rekey(string &$state): void {}
 #endif
 
+/** @sensitive-param $key */
 function sodium_crypto_shorthash(string $message, string $key): string {}
 
 function sodium_crypto_shorthash_keygen(): string {}
 
+/** @sensitive-param $secret_key */
 function sodium_crypto_sign(string $message, string $secret_key): string {}
```

Comment:
`$message`?

Reply:
No, this function signs the given message (similar to a GPG signature). You would usually only sign a message if you intend to _publish_ it later.

Reply:
Thanks!